Call them 2i2c managed JupyterHubs rather than hubs

yuvipanda opened the pull request:
https://github.com/2i2c-org/pilot-hubs/issues/314 talks
about the need to be very explicit in what we call the
service we provide. The word 'hub' is very overloaded.

2i2c managed infrastructure has an important property that our naming
should reflect.  It isn't a *new thing* we are developing, but a
deployment of an upstream, purely open source product. This means
our users can always:

1. Take their infrastructure from 2i2c, and run it themselves
   with help from the community.
2. Work with us to *train* their own staff to run their JupyterHub,
   so they can participate fully in running the infrastructure. This
   gives them organizational control over what is built, and they
   can walk away easier if needed.
3. Any improvements we make in terms of features built or
   operations smoothed will be made in upstream repositories, and

I think directly using the word JupyterHub is a good way to
communicate this.

Alternative suggestions:

1. just 'JupyterHub'
2. 2i2c JupyterHub
3. JupyterHub managed by 2i2c

I prefer (2) if needed.

Question is, how does this related to the Jupyter Trademark
policy (https://jupyter.org/governance/trademarks.html)?
Particularly:

> Any commercial use of the Jupyter trademarks in product or company
> names must be approved first by Project Jupyter. Some uses, like calling
> a company “The Jupyter Company,” or a product “Jupyter Hosting” or
> “Jupyter Cloud” will be refused. This is because they are overly broad,
> or confusing as to whether Jupyter is open source or commercial, or
> whether your product or organization is affiliated with or sponsored by
> Project Jupyter.

(from
https://jupyter.org/governance/trademarks.html#uses-that-always-require-approval)

So no matter what, I think this needs to go to the Jupyter steering
council. I personally believe that '2i2c managed JupyterHub' is fine,
but that's not for me to decide.

yuvipanda committed:
Call them 2i2c managed JupyterHubs rather than hubs

https://github.com/2i2c-org/pilot-hubs/issues/314 talks
about the need to be very explicit in what we call the
service we provide. The word 'hub' is very overloaded.

2i2c managed infrastructure has an important property that our naming
should reflect.  It isn't a *new thing* we are developing, but a
deployment of an upstream, purely open source product. This means
our users can always:

1. Take their infrastructure from 2i2c, and run it themselves
   with help from the community.
2. Work with us to *train* their own staff to run their JupyterHub,
   so they can participate fully in running the infrastructure. This
   gives them organizational control over what is built, and they
   can walk away easier if needed.
3. Any improvements we make in terms of features built or
   operations smoothed will be made in upstream repositories, and

I think directly using the word JupyterHub is a good way to
communicate this.

Alternative suggestions:

1. just 'JupyterHub'
2. 2i2c JupyterHub
3. JupyterHub managed by 2i2c

I prefer (2) if needed.

Question is, how does this related to the Jupyter Trademark
policy (https://jupyter.org/governance/trademarks.html)?
Particularly:

> Any commercial use of the Jupyter trademarks in product or company
> names must be approved first by Project Jupyter. Some uses, like calling
> a company “The Jupyter Company,” or a product “Jupyter Hosting” or
> “Jupyter Cloud” will be refused. This is because they are overly broad,
> or confusing as to whether Jupyter is open source or commercial, or
> whether your product or organization is affiliated with or sponsored by
> Project Jupyter.

(from
https://jupyter.org/governance/trademarks.html#uses-that-always-require-approval)

So no matter what, I think this needs to go to the Jupyter steering
council. I personally believe that '2i2c managed JupyterHub' is fine,
but that's not for me to decide.

choldgraf commented on content/infrastructure/features.md:
```diff
@@ -12,13 +12,13 @@ weight = 10
   icon_pack = "fas"
   name = "Provide a variety of open source interfaces"
-  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c Hub."
+  description = "Use a variety of user interfaces, including Jupyter Notebook, Jupyter Lab, RStudio, VSCode. If it can be run in a browser, it can run on a 2i2c managed JupyterHub."
```
maybe rather than saying stuff like "run on a 2i2c managed JupyterHub", we instead say stuff like "run on your JupyterHub"?

yuvipanda replied:
Yeah, I like that. Maybe 'your managed JupyterHub'? or is that redundant?

yuvipanda replied:
I've rewritten the list of features so they don't need to refer to the 'hub'. I also removed 'content' and added 'authentication' to the list, since I think that's more important. LMK what you think

yuvipanda committed:
Reword the 'features' page

- Remove 'distributing content', add 'authentication' instead
- Trim content size

yuvipanda committed:
Re-title infra intro section

choldgraf commented on config/_default/menus.toml:
```diff
@@ -4,7 +4,7 @@
 
 [[main]]
-  name = "2i2c Hubs"
+  name = "2i2c managed JupyterHubs"
```
```suggestion
  name = "Managed JupyterHubs"
```

I think we should just remove the name "2i2c" from here otherwise it's going to take a lot of navbar space, and I think it's implicit

choldgraf commented on content/infrastructure/features.md:
```diff
@@ -11,36 +11,36 @@ weight = 10
-  name = "Connect with your own online content"
-  description = "Provide notebooks, text files, and scripts for your hub's community through links that point to online repositories."
+  name = "Frictionless Authentication"
+  description = "Reduce onboarding friction by logging in your favorite authentication provider."
```
I think this is not quite a true statement, no? Don't we only provide real support for OAuth / github / orcid?

yuvipanda replied:
I think that's true for the pilot repos, but not true elsewhere. For example, the Farallon hub runs a *different flavor* of GitHub auth (without auth0, but with per-organization limits). UToronto uses Microsoft AAD. So I think this is overall an ok claim to make.

choldgraf commented on content/infrastructure/features.md:
```diff
@@ -11,36 +11,36 @@ weight = 10
-  name = "An environment for your community"
-  description = "2i2c Hubs provide data environments that have been designed for research and education, and can be customized for your community."
+  name = "Custom environments"
+  description = "Python? R? Julia? Use whatever custom data science stack you want."
```
Let's have the patterns similar across items. If we do `thing, thing2, thing3 - ` below, we should do the same pattern here instead of `thing? thing2? thing3`. Or vice-versa

yuvipanda replied:
Fixed

yuvipanda committed:
Shorten menu item name

yuvipanda committed:
Merge branch 'main' of github.com:2i2c-org/2i2c-org.github.io into nomenclature

yuvipanda committed:
Consistency fix

yuvipanda committed:
Minor grammatical fix

yuvipanda committed:
Merge branch 'main' into nomenclature

choldgraf committed:
Merge branch 'main' into nomenclature
